fortitude/src/rules/mod.rs

```diff
@@ -74,92 +70,89 @@ pub fn code_to_rule(category: Category, code: &str) -> Option<(RuleGroup, Rule)>
+        (Error, "000") => (RuleGroup::Stable, None, Default, error::ioerror::IoError),
+        (Error, "001") => (RuleGroup::Stable, Ast, Default, error::syntax_error::SyntaxError),
+        (Error, "011") => (RuleGroup::Stable, None, Default, error::allow_comments::InvalidRuleCodeOrName),
+
+        (Correctness, "001") => (RuleGroup::Preview, Ast, Default, correctness::select_default::MissingDefaultCase),
+        (Correctness, "011") => (RuleGroup::Preview, Ast, Default, correctness::trailing_backslash::TrailingBackslash),
+        (Correctness, "021") => (RuleGroup::Stable, Ast, Optional, correctness::kind_suffixes::NoRealSuffix),
+        (Correctness, "022") => (RuleGroup::Stable, Ast, Optional, correctness::implicit_kinds::ImplicitRealKind),
+        (Correctness, "031") => (RuleGroup::Preview, Ast, Optional, correctness::magic_numbers::MagicNumberInArraySize),
+        (Correctness, "032") => (RuleGroup::Preview, Ast, Optional, correctness::magic_numbers::MagicIoUnit),
+        (Correctness, "043") => (RuleGroup::Preview, Ast, Optional, correctness::missing_io_specifier::MissingActionSpecifier),
+        (Correctness, "051") => (RuleGroup::Stable, Ast, Default, correctness::implicit_typing::ImplicitTyping),
+        (Correctness, "052") => (RuleGroup::Stable, Ast, Default, correctness::implicit_typing::InterfaceImplicitTyping),
+        (Correctness, "053") => (RuleGroup::Preview, Ast, Default, correctness::implicit_typing::ImplicitExternalProcedures),
+        (Correctness, "061") => (RuleGroup::Stable, Ast, Default, correctness::intent::MissingIntent),
+        (Correctness, "071") => (RuleGroup::Stable, Ast, Default, correctness::assumed_size::AssumedSize),
+        (Correctness, "072") => (RuleGroup::Stable, Ast, Default, correctness::assumed_size::AssumedSizeCharacterIntent),
+        (Correctness, "081") => (RuleGroup::Stable, Ast, Default, correctness::init_decls::InitialisationInDeclaration),
+        (Correctness, "091") => (RuleGroup::Stable, Ast, Default, correctness::external::ExternalProcedure),
+        (Correctness, "101") => (RuleGroup::Preview, Ast, Default, correctness::derived_default_init::MissingDefaultPointerInitalisation),
+        (Correctness, "111") => (RuleGroup::Stable, Ast, Default, correctness::external_functions::ProcedureNotInModule),
+        (Correctness, "121") => (RuleGroup::Stable, Ast, Default, correctness::use_statements::UseAll),
+        (Correctness, "122") => (RuleGroup::Preview, Ast, Default, correctness::use_statements::MissingIntrinsic),
+        (Correctness, "131") => (RuleGroup::Preview, Ast, Default, correctness::accessibility_statements::MissingAccessibilityStatement),
+        (Correctness, "132") => (RuleGroup::Preview, Ast, Optional, correctness::accessibility_statements::DefaultPublicAccessibility),
+        (Correctness, "141") => (RuleGroup::Preview, Ast, Optional, correctness::include_statement::IncludeStatement),
```

Comment:
Perhaps better under modernization? MFE considers this more of a "deprecated" feature.

(I do see the argument for how it can be a correctness issue, but it does feel more like updating to use newer features of the language compared to just being a bug-prone thing.)